.github/workflows/release.yaml

```diff
@@ -25,178 +26,130 @@ env:
   ITCH_TARGET: the-bevy-flock/bevy-template
   # The organization or author that owns the rights to the game.
   OWNER: the-bevy-flock
+  # The path to the assets directory.
+  ASSETS_DIR: assets
   # Whether packages produced by this workflow should be uploaded to the Github release.
   UPLOAD_PACKAGES_TO_GITHUB_RELEASE: true
   # Before enabling LFS, please take a look at GitHub's documentation for costs and quota limits:
   # https://docs.github.com/en/repositories/working-with-files/managing-large-files/about-storage-and-bandwidth-usage
   USE_GIT_LFS: false
 
 jobs:
-  # Extract the version number from the pushed tag.
+  # Determine the version number for this workflow.
   get-version:
     runs-on: ubuntu-latest
     steps:
       - name: Get version number from tag
         id: tag
-        run: |
-          echo "tag=${GITHUB_REF#refs/tags/}" >> "${GITHUB_OUTPUT}"
+        run: echo "tag=${GITHUB_REF#refs/tags/}" >> "${GITHUB_OUTPUT}"
     outputs:
       version: ${{ inputs.version || steps.tag.outputs.tag }}
 
-  # Build for web.
+  # Build and package a release for web.
   build-for-web:
     runs-on: ubuntu-latest
     needs: get-version
     env:
       TARGET: wasm32-unknown-unknown
-      PROFILE: release
       PLATFORM: web
+      PACKAGE_EXT: .zip
       VERSION: ${{ needs.get-version.outputs.version }}
-
     steps:
-      - name: Set PACKAGE environment variable
-        run: |
-          echo "PACKAGE=${{ env.PACKAGE_NAME }}-${{ env.VERSION }}-${{ env.PLATFORM }}" >> "${GITHUB_ENV}"
-
-      - name: Set OUT_DIR environment variable
+      - name: Set up environment
         run: |
-          echo "OUT_DIR=build/${{ env.PACKAGE }}/${{ env.PACKAGE_NAME }}" >> "${GITHUB_ENV}"
+          echo 'PACKAGE=${{ env.PACKAGE_NAME }}-${{ env.PLATFORM }}' >> "${GITHUB_ENV}"
+          echo 'OUT_DIR=tmp/package/${{ env.PACKAGE_NAME }}' >> "${GITHUB_ENV}"
```

Comment:
This mixes both single quotes and double quotes. I would prefer the file prefer one and drop the other, except for special occasions.

- No quotes: similar to double quotes, but a chance a string will be evaluated as Bash.
- Single quotes: forces the contained expression to be a string, but does not support Bash `$VARIABLE` evaluation. (But `${{ expr }}` still works, because that's Github Actions.)
- Double quotes: forces the contained expression to be a string, and does support Bash `$VARIABLE` evaluation.

Reply:
This is intentional. The expressions like `${{ expr }}` get evaluated before the command is passed to the shell, so if the evaluated expression contains backslashes or `$` or other special characters, it would cause issues within double quotes. Single quotes make more sense.

Reply:
Ok, sounds good!